### ISSUE TITLE
[CNTOOLS] ./cntools.sh: line 2205: 6.50004e+10: syntax error: invalid arithmetic operator (error token is ".50004e+10")

### ISSUE THREAD

**Issue by alopezt16** [cardano-community/guild-operators#297]
CNTools **1.2**
**cardano-cli 1.14.2** - linux-x86_64 - ghc-8.6
git rev **924a6f7d8c2bdb1bf525be8b0d5626e440697b01**
genesis.json "systemStart": "**2020-06-28T21:00:00Z**",

**Steps:**
1. Open CNTOOLS
2. Go to Pools
3. Go to Show
4. Select a Pool

**Observed:**
The following error is displayed: 
`./cntools.sh: line 2205: 6.50004e+10: syntax error: invalid arithmetic operator (error token is ".50004e+10")`

**Expected:**
The pool information it's shown correctly.

**Comment by rdlrt** [comment]
Hi There,
Thanks for raising this issue. Could you confirm the version of CNTools you use (top bar in the home page), so that we can check the reference for that constant reported in exponential format?

**Comment by alopezt16** [comment]
>> CNTools **1.2** <<

**Comment by SmaugPool** [comment]
What is your `cardano-cli` version please (`cardano-cli --version`)?

**Comment by alopezt16** [comment]
**cardano-cli 1.14.2** - linux-x86_64 - ghc-8.6
git rev **924a6f7d8c2bdb1bf525be8b0d5626e440697b01**

  "systemStart": "**2020-06-28T21:00:00Z**",

**Comment by SmaugPool** [comment]
Thank you, last question, what is your pool id please?

**Comment by alopezt16** [comment]
Pool ID: **97197e4564902c53cc45a7c501f7a719588e87cd32a100301f7d18d2**

**Comment by SmaugPool** [comment]
If you are on telegram, please DM me @SmaugPool.

**Comment by SmaugPool** [comment]
If you are not, please:
```
$ cat /etc/os-release
$ lsb_release -a
$ awk --version
$ jq --version
```
Thank you.

**Comment by alopezt16** [comment]
The issue seems to be related to **Debian 10** and **awk**. @SmaugPool is currently looking into this.

**Comment by Scitz0** [comment]
Fix implemented in v2.0.0 that's to be released soon.
